_ios/tutorial.md

```diff
@@ -3,7 +3,7 @@ layout: learn
 description: How to use Blockstack on iOS Mobile
 permalink: /:collection/:path.html
 ---
-# iOS SDK Tutorial (Pre-release)
+# iOS DApps (Pre-release)
```

Comment:
Can also drop the "Pre-release" here as well

Reply:
Done